includes/class-sensei-learners-main.php

```diff
@@ -296,6 +297,7 @@ protected function get_row_data( $item ) {
 					array(
 						'title'        => '<strong><a class="row-title" href="' . esc_url( admin_url( 'user-edit.php?user_id=' . $user_activity->user_id ) ) . '" title="' . esc_attr( $a_title ) . '">' . esc_html( $title ) . '</a></strong>',
 						'date_started' => get_comment_meta( $user_activity->comment_ID, 'start', true ),
+            			'date_completed'  => ( 'complete' == $user_activity->comment_approved || 'graded' == $user_activity->comment_approved || 'passed' == $user_activity->comment_approved ) ? $user_activity->comment_date : '',
```

Comment:
`complete` is the only condition that needs to be checked here. `graded` and `passed` are lesson statuses and are not applicable to courses.

Reply:
+ fix whitespace errors.